DuckDuckGo/TabViewControllerBrowsingMenuExtension.swift

```diff
@@ -239,13 +239,25 @@ extension TabViewController {
         let image = UIImage(named: "MenuEmail")!
 
         return BrowsingMenuEntry.regular(name: title, image: image) { [weak self] in
-            self?.emailManager.getAliasIfNeededAndConsume { alias, _ in
+            guard let emailManager = self?.emailManager else { return }
+
+            var pixelParameters: [String: String] = [:]
+
+            if let cohort = emailManager.cohort {
+                pixelParameters[PixelParameters.emailCohort] = cohort
+            }
+            pixelParameters[PixelParameters.emailLastUsed] = emailManager.lastUseDate
```

Comment:
On the first invocation of this, the following URL was sent:

```
https://improving.duckduckgo.com/t/email_filled_random_ios_phone?cohort=internal_beta&duck_address_last_used=&test=1
```

Are there any implications around sending an empty parameter for `duck_address_last_used`?

Reply:
No, it's supposed to be empty according to the spec (see comments in Asana task).